pkgs/phps.nix

```diff
@@ -45,6 +45,22 @@ let
           else
             super.extensions.mysqlnd;
 
+        oci8 = super.extensions.oci8.overrideAttrs (attrs: (
```

Comment:
I would prefer using the package from Nixpkgs for all PHP versions where it is compatible.

Reply:
Right. that makes sense. I'll fix that up :+1: